core-api/src/main/java/com/optimizely/ab/config/ProjectConfig.java

```diff
@@ -78,19 +79,20 @@ public String toString() {
 
     public ProjectConfig(String accountId, String projectId, String version, String revision, List<Group> groups,
                          List<Experiment> experiments, List<Attribute> attributes, List<EventType> eventType,
-                         List<Audience> audiences) {
-        this(accountId, projectId, version, revision, groups, experiments, attributes, eventType, audiences,
+                         List<Audience> audiences, boolean anonymizeIP) {
```

Comment:
Will do. This will ensure that we always have it initialized to `false` unless explicitly passed in to the constructor.